test/bffs-test.js

```diff
@@ -40,32 +39,40 @@ describe('bffs', function () {
   var bffs;
 
   //
-  // Setup datastar and the models before anything else
+  // Setup database and the models before anything else
+  // S3 configuration for both environments is identical for
+  // tests, only single bucket needs to be prepared.
   //
   before(function (next) {
-    datastar = new Datastar(config);
-    models = wrhs(datastar);
-    redis = new Redis()
-      .on('error', err => console.error(err));
+    this.timeout(60000);
 
-    datastar.connect(function (err) {
-      if (err) return next(err);
+    const dynamoDriver = new DynamoDB(config.dynamodb);
+
+    dynamo.dynamoDriver(dynamoDriver);
+    models = wrhs(dynamo);
+    redis = new Redis().on('error', console.error); // eslint-disable-line no-console
```

Comment:
Do we even have fuctionality that actually still stores in Redis, other than a few isolated specific tests?

Reply:
@Swaagie its mostly legacy

Reply:
Should I just remove it now that we release a new major anyways? I think our current understanding is that builds take so long that we don't need a quick key/value store

Reply:
Made a new ticket to remove the essentially dead code